src/main/resources/application.yml

```diff
@@ -2,7 +2,7 @@ spring:
   datasource:
     url: jdbc:postgresql://localhost:5432/betting
     username: postgres
-    password: admin
+    password: 1234
```

Comment:
Don't commit local database/any password, please change to what it was before (let me know when you get to this, I can show you a trick that helps with this)